mne_bids/utils.py

```diff
@@ -485,8 +484,12 @@ def copyfile_eeglab(src, dest):
 
     # Extract matlab struct "EEG" from EEGLAB file
     # if the data field is a string, it points to a .fdt file in src dir
-    eeg = _check_load_mat(src, None)
-    if isinstance(eeg['data'], string_types):
+    mat = loadmat(src, matlab_compatible=True)
+    if 'EEG' not in mat:
+        raise ValueError('Could not find "EEG" field in {}'.format(src))
+    eeg = mat['EEG']
+    data = eeg[0][0]['data']
+    if all([item in data[0, -4:] for item in '.fdt']):
```

Comment:
I'm a bit lost here. Looks like it passed my attention earlier. If you raise a ValueError, what are the following lines doing?

Reply:
also can you be more explicit below what issues you are having doing the round trip?

Reply:
> I'm a bit lost here. Looks like it passed my attention earlier. If you raise a ValueError, what are the following lines doing?

I am not sure I am getting your meaning, but I raise a ValueError for `.mat` files that do not have an EEG field ... if such a field is present, I continue as planned

regarding the roundtrip, I agree with you --> let's do it in a new PR. I will see what I can do (also with regards too your comment in the bep021 spec)

Reply:
will these lines every be reached in your code: https://github.com/mne-tools/mne-bids/blob/master/mne_bids/utils.py#L499-L519 ?

Reply:
No, currently they will never be reached because they are not working (yet). I left them in there as a good place to start when tackling #88 ... reconsidering, I could have included that code in the issue text of #88 and left it out of the code.

Reply:
yes, I think for the future, we should do that. Never include code that you plan to use later ...